README.md

```diff
@@ -29,95 +33,319 @@ are well established for RSE-ops, and which require further development. We can
 
+> Yeah agree with this. Cloud resources/CI runners are really needed. Centers also have a hard time
+> making arguments to justify cycles on test clusters (though we do have dedicated ones
+> internally). Supporting teams who need to test on LC-relevant machines for projects that live on
+> GitHub was an ISCP project of Greg's this year -- we built a cluster in the green zone to support
+> it.  We should talk about this more -- this was an ISCP project that I proposed.
+
+
 ### What about scaling?
 
 High performance computing was designed for scaling, but it must be done intentionally. By way of job managers, MPI programming, and
-extreme parallelism [^8], it's relatively trivial to scale a task on an HPC cluster. However, the user has to know what they are doing. A related issue is testing at scale. As tests need to be run frequently and across many different architectures, the incentive structure is arguably not there to maintain such a testing framework. This is in contrast to some Cloud Native tools that handle scaling or scaled testing for the user. This suggests that the greatest challenges are not related to the technologies, but educating the user base and RSEs how to properly use them. There is a huge opportunity for development of RSE-ops tools that can better automate this scaling.
+extreme parallelism [^8], it's relatively trivial to scale a task on an HPC cluster. However, the user has to know what they are doing.
+
+> Don't these two sentences contradict each other?
+
+A related issue is testing at scale. As tests need to be run frequently and across many different architectures, the incentive structure is arguably not there to maintain such a testing framework.
 
+> we really need an HPC test harness. There are attempts at this for app testing (3 versions of a
+> tool called ATS internally) and there ar things like Pavilion2
+> (https://github.com/hpc/pavilion2), ReFrame (https://github.com/eth-cscs/reframe) and BuildTest
+> (1 developer/user who you know) for facility testing but there isn't anything I can see here that
```

Comment:
haha, hey I was just trying to help with BuildTest. I tried to make the structure of the code more logical / easier to develop, but I wouldn't champion it as a library I'd use.

Reply:
yeah I dunno if we need to mention it :)